tutorials/13_diagnosis_sbc.ipynb

```diff
@@ -0,0 +1,1897 @@
+      "text/plain": [
+       "<Figure size 576x360 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from sbi.analysis.plot import sbc_rank_plot\n",
+    "\n",
+    "f, ax = sbc_rank_plot(ranks, 1_000, plot_type = \"hist\", num_bins=30)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "The two plots show only visualize the histogram across the `ranks` (here depicted in red). Highlighted in grey is the 95% confidence interval of a uniform distribution given the number of samples provided. In plain english, the majority of red bars should reach the grey area of the confidence interval. \n",
```

Comment:
"The two plots visualize the distributions of the ranks for each posterior dimension (for each parameter) as histograms."

Reply:
it's the 99% confidence interval of histograms we would expect under uniformity.
"In plain english, for a uniform distribution we would expect only "one in a hundred" histogram bars to lie outside the gray band. "

Reply:
7ba0ece